git/homework.py

```diff
@@ -116,18 +138,22 @@ def remove_from_list_all_negative_numbers(data: List[int]) -> list:
         remove_from_list_all_negative_numbers([1, 5, -7, 8, -1])
         >>> [1, 5, 8]
     """
-    pass
+    data = [i for i in data if i >= 0]
+    return data
 
```

Comment:
could combine lines 141 and 142. You don't need to create separate variable for list compr.